pytaxonkit.py

```diff
@@ -951,6 +956,11 @@ def lca(ids, multi=False, skip_deleted=False, skip_unfound=False, threads=None,
     out, err = proc.communicate(input=idstring)
     if proc.returncode != 0:
         raise TaxonKitCLIError(err)  # pragma: no cover
+    if out.strip() == '':
+        if multi:
+            return [None] * len(ids)
+        else:
+            return None
```

Comment:
This change fixes the parsing bug that resulted when an input consists entirely of deleted an unfound nodes.

Reply:
Can confirm this changes prevents the parsing error. Using the previous example that caused it to fail
```
>>> pytaxonkit.lca([2912032, 2901441, 2907121], skip_deleted=True, skip_unfound=True)
911341
```